website/content/en/docs/User Guide/Learn Cloud Operations/cloud_error_reporting.md

```diff
@@ -37,4 +38,50 @@ Click **View Logs** for one of the samples to see the log messages that match th
 
 You can expand any of the messages that matches the filter to see the full stack trace:
 
-![image](/docs/images/user-guide/35-logdet.png)
\ No newline at end of file
+![image](/docs/images/user-guide/35-logdet.png)
+
+#### Errors Manufacturing 
+
+There are several ways in which you can experiment with Error Reporting tool and manufacture errors that will be reported and displayed in the UI. For the purpose of this demonstration we will use 2 tools that are coming with Cloud Operations Sandbox: Load Generator and SRE Recipes to simulate a situation that `Sandbox break`.
+
+To simulate requests using the load generator we can use the UI or `sandboxctl`
```

Comment:
1) "or the `sandboxctl` command line tool", in case they aren't familiar
2) I'd probably suggest not mentioning the UI here if the tutorial is just showing the command line. Unless we can link to a doc that explains how to use the UI?

Reply:
At the second step I'm using the UI, I added double new line - let me know if now its clearer `Another way to `break the service` is to use the load generator to overload the service with too many requests.`